boards/holybro/kakutef7/default.px4board

```diff
@@ -17,11 +17,11 @@ CONFIG_DRIVERS_OSD_ATXXXX=y
 CONFIG_DRIVERS_PWM_OUT=y
 CONFIG_DRIVERS_RC_INPUT=y
 CONFIG_DRIVERS_TELEMETRY_FRSKY_TELEMETRY=y
-CONFIG_MODULES_ATTITUDE_ESTIMATOR_Q=y
 CONFIG_MODULES_BATTERY_STATUS=y
 CONFIG_MODULES_COMMANDER=y
 CONFIG_MODULES_CONTROL_ALLOCATOR=y
 CONFIG_MODULES_DATAMAN=y
+CONFIG_MODULES_EKF2=y
```

Comment:
Can you disable these ekf parts: https://github.com/PX4/PX4-Autopilot/blob/main/boards/px4/fmu-v2/default.px4board#L26-L31?

Reply:
Ok, even if a GPS might be connected potentially? And would you do it for all Kakutes or just the F7?

Reply:
GPS will still work. We can do it for all of them as they're generally low on flash.

Reply:
Alright, will do. Thanks.

Reply:
@bkueng it looks like there is no way to fit EKF2 onto 1MB flash on the KakuteF7. Any ideas what to do? Retire it, or going back to Q?

Reply:
Let's keep Q then.